sdcm/cluster_gce.py

```diff
@@ -297,9 +298,15 @@ def _get_disks_struct(self, name, dc_idx):
     def _create_instance(self, node_index, dc_idx, spot=False, enable_auto_bootstrap=False):
         def set_tags_as_labels(_instance):
             self.log.debug(f"Expected tags are {self.tags}")
-            lower_tags = dict((k.lower(), v.lower().replace('.', '-')) for k, v in self.tags.items())
-            self.log.debug(f"Lower tags are {lower_tags}")
-            self._gce_services[dc_idx].ex_set_node_labels(_instance, lower_tags)
+            # https://cloud.google.com/resource-manager/docs/tags/tags-creating-and-managing#adding_tag_values
+            regex_key = re.compile(r'[^a-z-_]')
```

Comment:
adding the number would make it a bit more reasonable:
```suggestion
            regex_key = re.compile(r'[^a-z0-9-_]')
```